contracts/schemes/Auction4Reputation.sol

```diff
@@ -121,6 +122,7 @@ contract Auction4Reputation is Ownable {
         address(token).safeTransferFrom(msg.sender, address(this), _amount);
         // solhint-disable-next-line not-rely-on-time
         auctionId = (now - auctionsStartTime) / auctionPeriod;
+        require(auctionId == _auctionId, "false auction id");
```

Comment:
More clear revert string would be "auction has completed"

Or if that presumes too much, then "auction is not active"